pack_and_terraform.bash

```diff
@@ -140,19 +147,25 @@ if echo "$RUNTF" | grep -iq "^y"; then
 	fi
 
 	export LC_CTYPE=C
-	HMAC=`cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | base64`
+	HMAC=`cat /dev/urandom | env LC_CTYPE=C | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | base64`
```

Comment:
This won't work because you are passing the output of `env` to `tr` so you don't get an entropy